static/app/bootstrap/exportGlobals.tsx

```diff
@@ -63,15 +49,12 @@ globals.SentryApp = {
   },
 
   // The following components are used in legacy django HTML views
-  passwordStrength: {load: loadPasswordStrength},
-  U2fSign: require('app/components/u2f/u2fsign').default,
   ConfigStore: require('app/stores/configStore').default,
-  SystemAlerts: require('app/views/app/systemAlerts').default,
-  Indicators: require('app/components/indicators').default,
-  SetupWizard: require('app/components/setupWizard').default,
   HookStore: require('app/stores/hookStore').default,
   Modal: require('app/actionCreators/modal'),
 };
 
 // Make globals available on the window object
 Object.keys(globals).forEach(name => (window[name] = globals[name]));
```

Comment:
Maybe this should no longer be a side effect in here now that this module is exporting something.

Reply:
We need to keep this here for now for compatibility (e.g. custom plugins)

Reply:
Yeah, I'm just suggesting moving the side effect somewhere more explicit. Instead of having the module do this on import

Reply:
Ah - I think it's fine to keep this in here for now, ideally this gets removed completely.